common/daemon.c

```diff
@@ -153,7 +170,11 @@ void daemon_setup(const char *argv0,
 
 	/* We handle write returning errors! */
 	signal(SIGPIPE, SIG_IGN);
+
+	/* We set up Wally, the bitcoin wallet lib */
+	wally_tal_ctx = notleak_with_children(tal(NULL, char));
```

Comment:
I think we ran into problems later with this,so chose to use a '_notleak' tal label instead?